lib/pbench/server/api/auth.py

```diff
@@ -79,7 +79,7 @@ def get_auth_token(self, logger):
                 )
                 abort(
                     401,
-                    message="Malformed Authorization header, request auth needs bearer token: Bearer <session_token>",
+                    message="Malformed Authorization header, request authorization header needs bearer token: Bearer <session_token>",
```

Comment:
Perhaps just remove `authorization header` since those two exact words are written before `request`.  Avoids invoking the Department of the Redundancy Department.